docs/Deploy/setup-azuredevops.md

```diff
@@ -1,53 +1,136 @@
+*Create* two new pipelines, selecting the existing files `.azure-pipelines/azops-pull.yml` & `.azure-pipelines/azops-push.yml`.
+It is a good practice to name these pipelines `AzOps - Pull` and `AzOps - Push` respectively (in both the YAML file, and the pipeline itself after you create it).
 
-2. Import the repository
-   Within the Azure DevOps project, import the [Enterprise-Scale](https://github.com/Azure/Enterprise-Scale) repository from GitHub. Instructions are [here](https://docs.microsoft.com/azure/devops/repos/git/import-git-repository).
+![Creating a pipeline](../media/ado-pipeline-create.png)
 
-### Configure the pipelines
+When creating the pipelines, define a new secret variable to each of the pipelines:
 
-Add two new pipelines, selecting the existing files `.azure-pipelines/azops-pull.yml` & `.azure-pipelines/azops-push.yml`. It is a good practice to name these pipelines `AzOps - Pull` & `AzOps - Push`.
-When adding the pipelines define a new secret variable to each of the pipelines:
+* AZURE_CREDENTIALS - Set the value to the *escaped JSON* string from the script at the beginning of this article
 
-* AZURE_CREDENTIALS - Set the value to the JSON string created by the steps in the [GitHub - Setup Guide](setup-github.md). (escape the quotes in the json string from `"` to  `\"`)
+![Pipeline variable](../media/ado-pipeline-variable.png)
 
-_Optional step to configure approval steps_
-The `AzOps - Push` push pipeline creates an ADO environment on this environment select `Approvals and checks` and configure an Approval for a user or group.
+## 4. Configure the Environment Approvers
 
-### Configure repository permissions
+In Environments, open `AzureEnv` and select `Approvals and checks`, then configure an Approval for a user or group.
 
-The build service account `<Project> Build Service (<Organization>)` must have the following permissions on the repository:
+![Environment approval](../media/ado-env-approval.png)
+
+## 5. Configure repository permissions
+
+The build service account must have the following permissions on the repository:
 
 * `<Project>\Contributors`
 
-### Configure branch policies
+In the *Repository* - *Manage Repository* - *Permissions* screen, ensure that in the Repository permissions that the Contributors group has the `Force Push` permission set to Allow.  
+
+![Force Push](../media/ado-repo-forcepush.png)
+
+Whilst on the Repository Permissions screen, find the Build User in the list and copy their name for the next step.
+
+![Build service username](../media/ado-repo-buildservice.png)
+
+Finally, add the user to the Contributor group in *Project Settings* - *Permissions* - *Contributors* - *Members*.
+
+![Permissions](../media/ado-permissions-group.png)
+
+### 6. Configure repository branch policies
```

Comment:
Lets use this as an optional step. Have seen company doing to push only up on approval on the environment. That is where we deviate from the GH action flow

Reply:
The whole of Step 6? 
I was under the impression you need it to trigger the AzOps push.  I'll try in my ESIAB environment.

Reply:
Have tried in my ADO.
The AzOps push will only run as part of a PR, which is done with a branch build policy.
Happy to catch up on this @hjscherer

Reply:
yes, lets do it. With the environment in place the branch policy validation isn't required.